sugarscape.py

```diff
@@ -893,50 +911,41 @@ def updateRuntimeStats(self):
             agentDiseaseDeaths += 1 if agent.causeOfDeath == "disease" else 0
```

Comment:
Modify this to `agentDiseaseDeaths += 1 if agent.diseaseDeath == True else 0` since there is no way to detect if an agent dies from disease.

Reply:
Leaving a note here to include this in the enhanced disease mechanics in your PR. This merge is going to cause conflicts with that since there are so many (good) changes in your PR.